### DISCUSSION TITLE
Help with Linux installation

### DISCUSSION THREAD

**Discussion by ivanced09** [bmaltais/kohya_ss#666] [Q&A]
People, I am installing the GUI on ubuntu because from what I see it is the only way to train with multiple gpus simultaneously (i have two 3060 12gb), I find several problems and doubts:

![Captura desde 2023-04-25 11-33-48](https://user-images.githubusercontent.com/113315177/234319970-2d14a86a-044b-4e9c-8df9-18389f19dfef.png)
![Captura desde 2023-04-25 11-34-25](https://user-images.githubusercontent.com/113315177/234319985-312cea49-caa8-4139-8fe6-b09844f65aaa.png)

How to open "accelerate config" in linux? Since it does not appear at any time to configure it and from what I read part of the problem comes from that side, After another question that I have, are the nvidia drivers that I use in linux relevant? From what I see, there are the open drivers that come with the same OS and there are also the closed drivers from the manufacturer. Is this relevant to this?

**Comment by virt-10** [comment]
Yes, the drivers you use are important, you want to use the closed sourced Nvidia drivers.
Looks like pytorch is installed without cuda **and/or** you do not have CUDA and CUDNN installed in your system

#to check for cuda and cudnn
nvcc -V

#should return something like this

> nvcc: NVIDIA (R) Cuda compiler driver
> Copyright (c) 2005-2022 NVIDIA Corporation
> Built on Wed_Sep_21_10:33:58_PDT_2022
> Cuda compilation tools, release 11.8, V11.8.89
> Build cuda_11.8.r11.8/compiler.31833905_0

To open accaccelerate config just run these commands in the kohya folder

#activates venv
source venv/bin/activate

#runs accelerate config
accelerate config

#runs accelerate test
accelerate test
